### DISCUSSION TITLE
Using import to make multiple updates without dupes or deleting.

### DISCUSSION THREAD

**Discussion by michaelvenzor** [code4recovery/12-step-meeting-list#558] [Q&A] [answered]
I want to add website URLs to many of my meetings. So what I'd like to do is:

1. Download all the meetings in a CSV. Done.
2. Add the URL to the "Website" column. No problem.
3. Re-upload the CSV and commit all the updates in one step. This one is tricky.

When I want to upload a CSV, I have three options:

1. don't delete anything - This is no good because it creates duplicates of all my meetings and I have to then delete all the dupes.
2. delete only the meetings, locations, and groups for the regions present in this CSV - I don't know what this means but it looks like it's going to delete all the meetings in a certain region.
3. delete all meetings, locations, groups, districts, and regions - This one is bad. Deleted everything. Luckily, I was able to put it all back with the CSV.

Is there a way to just make updates with having duplicates created? Or do I need to delete everything and then replace everything with the downloaded CSV? (The latter option is terrifying.)

Thanks!

**Comment by tech2serve** [comment, marked as answer]
Hi @michaelvenzor,

Option three is probably what you want to be using, and so we need to figure out what you experienced. If you don’t have a dev site, you might want to get one as this really reduced the stress of testing new approaches :-)

Option three does indeed delete existing meetings, but it replaces them with everything in your csv file. So unless you’ve made a mistake in the csv file, it *should* import with the previous data (from your download) and contain the new website URLs you added. 

If this didn’t happened, what errors did you see? Try again with the developer console open so you can see any errors thrown there. Given that we have a lot of users who routinely import their data in this fashion, I’m confident you can get this working on your site. We just need to figure out what went wrong.
